site/public/js/polls.js

```diff
@@ -106,6 +106,106 @@ function updateDropdownStates(curr_state, cookie_key) {
     Cookies.set(cookie_key, !curr_state, { expires: expiration_date, path: '/' });
 }
 
+function togglePollFormOptions() {
+    const correct_options = $('.correct-box');
+
+    correct_options.each(function () {
+        $(this).prop('checked', $('#toggle-all').prop('checked'));
+    });
+}
+
+function validateCustomResponse() {
+    const custom_response = document.querySelector('.custom-poll-response');
+    const custom_response_submit = document.querySelector('.custom-response-submit');
```

Comment:
It would be better to use jQuery for this.

Reply:
[38d4493](https://github.com/Submitty/Submitty/pull/10197/commits/38d449357404e5e5f0d24b6e6bfccc6c7e33d678)